src/test/java/nextstep/subway/line/domain/SectionsTest.java

```diff
@@ -195,10 +198,38 @@ public void alreadyExistStation() {
 
 
     /**
-     * 기존 : 영등포역 --10--> 신길역 --7--> 대방역
-     * 신규 : 영등포역 --17--> 대방역
-     */
-    @DisplayName("구간이 두개 이상일 경우, 가운역을 제거한다.")
+     *  기존 : 영등포역 --10--> 신길역 --7--> 대방역 --3--> 노량진역 --5--> 용산역
+     *  신규 : 영등포역 --10--> 신길역 --10--> 노량진역 --8--> 용산역
```

Comment:
지금도 충분히 좋구요 👍 
요런식으로 표현하면 어느 역이 제거되는지 좀 더 편하게 알 수 있을 것 같습니다! :)
```suggestion
     *  기존 : 영등포역 --10--> 신길역 --7--> 대방역 --3--> 노량진역 --5--> 용산역
     *  신규 : 영등포역 --10--> 신길역 --------10 -------> 노량진역 --8--> 용산역
```